websites/S/Sorryfield/metadata.json

```
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "741973166364164099",
+		"name": "ihah"
+	},
+	"service": "Sorryfield",
+	"altnames": [
+		"쏘리들"
+	],
+	"description": {
+		"en": "Sorryfield is a website which listing karaoke songs. And you can play rhythm game Java!",
+		"ko_KR": "쏘리들은 노래방 곡 목록 사이트입니다. 그리고 리듬 게임 자바!를 플레이 할 수 있습니다."
+	},
+	"url": "sorry.daldal.so",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/TxuMfTd.png",
```

There is a more high quality version available.
```suggestion
	"logo": "https://i.imgur.com/Rysn3ig.png",
```